doc/nrf/releases/release-notes-2.1.0.rst

```diff
@@ -0,0 +1,1063 @@
+
+  * Added configuration for the nRF5340 target.
+  * Fixed advertising start on the nRF5340 target with the Zephyr LL controller.
+    Previously, it was not possible to start advertising, because the :kconfig:option:`CONFIG_BT_EXT_ADV` option was disabled for the Zephyr LL controller.
+
+* :ref:`bluetooth_central_hr_coded` sample:
+
+  * Added configuration for the nRF5340 target.
+  * Fixed scanning start on the nRF5340 target with the Zephyr LL controller.
+    Previously, it was not possible to start scanning, because the :kconfig:option:`CONFIG_BT_EXT_ADV` option was disabled for the Zephyr LL controller.
+
+* :ref:`peripheral_fast_pair` sample:
+
+  * Added:
+
+    * Possibility of toggling between show and hide UI indication in the Fast Pair not discoverable advertising.
+    * Automatic switching to the not discoverable advertising with the show UI indication mode after 10 minutes of discoverable advertising.
+    * Automatic switching from discoverable advertising to the not discoverable advertising with the show UI indication mode after a Bluetooth Central successfully pairs.
+
+* :ref:`bluetooth_direction_finding_connectionless_tx` sample:
+
+  * Fixed a build error related to the missing :kconfig:option:`CONFIG_BT_DF_CONNECTIONLESS_CTE_TX` Kconfig option.
+    The option has been added and set to ``y`` in the sample's :file:`prj.conf` file.
+
+* :ref:`ble_throughput` sample:
+
+  * Fixed peer throughput calculations.
+    These were too low because the total transfer time incorrectly included 500ms delay without including the actual transfer.
+  * Optimized throughput speed by increasing MTU to 498 and using the maximum connection event time.
+
+* :ref:`bluetooth_direction_finding_central` sample:
+
+  * Added devicetree overlay file for the nRF5340 application core that configures GPIO pin forwarding.
+    This enables the radio peripheral's Direction Finding Extension for antenna switching.
+
+* :ref:`bluetooth_direction_finding_connectionless_rx` sample:
+
+  * Added devicetree overlay file for the nRF5340 application core that configures GPIO pin forwarding.
+    This enables the radio peripheral's Direction Finding Extension for antenna switching.
+
+* :ref:`bluetooth_direction_finding_connectionless_tx` sample:
+
+  * Added devicetree overlay file for the nRF5340 application core that configures GPIO pin forwarding.
+    This enables the radio peripheral's Direction Finding Extension for antenna switching.
+
+* :ref:`bluetooth_direction_finding_peripheral` sample:
+
+  * Added devicetree overlay file for the nRF5340 application core that configures GPIO pin forwarding.
+    This enables the radio peripheral's Direction Finding Extension for antenna switching.
+
+Bluetooth mesh samples
+----------------------
+
+* :ref:`bluetooth_mesh_light_lc` sample:
+
+  * Added overlay file with support for storing data with :ref:`emds_readme`.
+    Also changed the sample to restore Light state after power-down.
+
+nRF9160 samples
```

Comment:
Perhaps here we could add a note about TF-M output being unavailable when Modem tracing is enabled?

Reply:
Right, forgot to add it. Let me do it now.